src/healthkit.cs

```diff
@@ -1134,6 +1197,28 @@ interface HKQuery {
 		[Static]
 		[Export ("predicateForActivitySummariesBetweenStartDateComponents:endDateComponents:")]
 		NSPredicate GetPredicateForActivitySummariesBetween (NSDateComponents startDateComponents, NSDateComponents endDateComponents);
+
+
+		// @interface HKClinicalRecordPredicates (HKQuery)
+		[NoWatch, iOS (12,0)]
+		[Static, Internal]
+		[Export ("predicateForClinicalRecordsWithFHIRResourceType:")]
+		NSPredicate GetPredicate (NSString resourceType);
```

Comment:
All the other predicateFor* selectors in this class are bound as `GetPredicateFor*`, so I think the same applies here: `GetPredicateForClinicalRecords` (and the overloads below as well).